docs/operations/runbooks/downtime_incident_management.md

```diff
@@ -16,12 +16,14 @@ The following set of rules should be followed while an incident is in progress.
   - If downtime occurs outside of working hours, team members who are off for the day may still be pinged and called but are not required to join if unavailable to do so.
 - Uncomment the [banner on get.gov](https://github.com/cisagov/get.gov/blob/0365d3d34b041cc9353497b2b5f81b6ab7fe75a9/_includes/header.html#L9), so it is transparent to users that we know about the issue on manage.get.gov.
   - Designers or Developers should be able to make this change; if designers are online and can help with this task, that will allow developers to focus on fixing the bug.
+- If the issue persists for three hours or more, follow the [instructions for enabling/disabling a redirect to get.gov](https://docs.google.com/document/d/1PiWXpjBzbiKsSYqEo9Rkl72HMytMp7zTte9CI-vvwYw/edit)
```

Comment:
Note that this is staying in the google doc so we do not have the explicit steps we take here be publicly known

Reply:
I don't think so. We reference a few internal docs, like the onboarding doc, in our md files and we don't ever say "internal use only" I think it's somewhat implied when a user clicks it and sees their access is restricted. That said, very good question!